app/models/allocation.rb

```diff
@@ -51,6 +65,10 @@ def initialize(job:, nodes:)
     end
   end
 
+  def valid?
+    true
+  end
```

Comment:
Why do we have this?

Reply:
https://github.com/openflighthpc/flight-scheduler-controller/pull/48/commits/85214b58f53741d7297b034b3e2a46bc5bedd7b4